corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js

```diff
@@ -54,6 +54,8 @@ hqDefine("cloudcare/js/formplayer/menus/collections", function () {
             'multiSelect',
             'maxSelectValue',
             'hasDetails',
+            'entities',
```

Comment:
I don't think it's necessary to add 'entities' here.

Reply:
[357737e](https://github.com/dimagi/commcare-hq/pull/33157/commits/357737eea68bc1b6753b45c69c8fab7df631f49f)